components/unzipped/dashboard/Kanban/MyProjectsLists.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import {
+    Accordion,
+    AccordionSummary,
+    AccordionDetails,
+    Typography,
+    List,
+    ListItem,
+    ListItemIcon,
+    ListItemText
+} from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import ListIcon from '@mui/icons-material/List';
+import Button from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import { styled } from '@mui/system';
+import Menu from '@mui/material/Menu';
+import { SpanStyled } from './AssignedToList';
+import DndFilterIcons from '../../../icons/dndFilterIcon';
+
+const CustomButton = styled(Button)(({ theme }) => ({
+    marginTop: "13px",
+    color: "#000",
+    backgroundColor: "transparent !important",
+    fontSize: "17px",
+    fontWeight: "500",
+    textTransform: "none",
+    '&:active': {
+        backgroundColor: "transparent !important",
+    },
+    '&:focus': {
+        backgroundColor: "transparent !important",
+    },
+    '&:hover': {
+        backgroundColor: "transparent !important",
+    },
+}));
+
+
+const Container = styled('div')({
+    paddingLeft: '20px',
+    width: '100%',
+});
+
+const StyledListItem = styled(ListItem)({
+    margin: 0,
+    padding: '4px 0',
+});
+
+const BlueListIcon = styled(ListIcon)({
+    color: '#2F76FF',
+});
+
+const StyledMenu = styled((props) => (
+    <Menu
+        elevation={0}
+        anchorOrigin={{
+            vertical: 'bottom',
+            horizontal: 'right',
+        }}
+        transformOrigin={{
+            vertical: 'top',
+            horizontal: 'right',
+        }}
+        {...props}
+    />
+))(({ theme }) => ({
+
+    '& .MuiPaper-root': {
+        borderRadius: 6,
+        marginTop: "0px",
+        minWidth: 500,
+        color: "#737373",
+        '& .MuiMenu-list': {
+            padding: '2px 0',
+        },
+    },
+    '& .MuiModal-root-MuiPopover-root-MuiMenu-root': { left: "175px" }
+}));
+
+
+
+const projects = [
+    {
+        name: 'Unzipped',
```

Comment:
Projects are hard coded here. They should come from the logged in user's projects that they have started, or that they have an open contract for.